plugins/gatsby-source-data-dictionary/gatsby-node.js

```diff
@@ -0,0 +1,107 @@
+const uniq = (arr) => [...new Set(arr)];
+
+const getFileRelativePath = (absolutePath) =>
+  absolutePath.replace(`${process.cwd()}/`, '');
+
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions;
+
+  createTypes(`
+    type DataDictionaryEvent implements Node {
+      name: String!
+      definition: MarkdownRemark! @link
+      dataSources: [String!]!
+    }
+
+    type DataDictionaryAttribute implements Node {
+      name: String!
+      definition: MarkdownRemark! @link
```

Comment:
little confused by `@link` does. Tried reading the documentation on foreign key relations, but I'm not sure how will this relate the definition